Woof/WoofTests/ViewModel/OwnerProfileViewModelTests/SitterMainTabViewModelTests.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+
+final class SitterMainTabViewModelTests: XCTestCase {
+    func testSitterMainTabViewModelExistsInAPI() {
+        SitterMainTabViewModel()
```

Comment:
Do we need this test, taking into account we have the `testResertCurrentSitterRoleMethodExistsInAPI` ?

Reply:
You are right. Thanks for the attention! This test is useless. I removed it from both files.